pwndbg/lib/abi.py

```diff
@@ -77,8 +77,8 @@ class SigreturnABI(SyscallABI):
 linux_arm_syscall = SyscallABI(["r7", "r0", "r1", "r2", "r3", "r4", "r5", "r6"], 4, 0)
 linux_aarch64_syscall = SyscallABI(["x8", "x0", "x1", "x2", "x3", "x4", "x5", "x6"], 16, 0)
 linux_mips_syscall = SyscallABI(["$v0", "$a0", "$a1", "$a2", "$a3"], 4, 0)
-linux_ppc_syscall = ABI(["r0", "r3", "r4", "r5", "r6", "r7", "r8", "r9"], 4, 0)
-linux_ppc64_syscall = ABI(["r0", "r3", "r4", "r5", "r6", "r7", "r8", "r9"], 8, 0)
+linux_ppc_syscall = SyscallABI(["r0", "r3", "r4", "r5", "r6", "r7", "r8"], 4, 0)
```

Comment:
```suggestion
linux_ppc_syscall = SyscallABI(["r0", "r3", "r4", "r5", "r6", "r7", "r8", "r9"], 4, 0)
```

Suggesting this according to the syscall table from `man 2 syscall`